--- a/arch/arm/boot/dts/overlays/mm610x-spi-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mm610x-spi-overlay.dts
@@ -17,8 +17,8 @@
 				compatible = "morse,mm610x-spi";
 				reg = <0>;	/* CE0 */
 				reset-gpios = <&gpio 17 0>;
-				power-gpios = <&gpio 3 0>,
-				              <&gpio 7 0>;
+				power-gpios = <&gpio 23 0>,
+				              <&gpio 24 0>;
 				spi-irq-gpios = <&gpio 5 0>;
 				spi-max-frequency = <50000000>;
 				status = "okay";
--- a/arch/arm/boot/dts/overlays/mm810x-spi-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mm810x-spi-overlay.dts
@@ -17,8 +17,8 @@
 				compatible = "morse,mm810x-spi";
 				reg = <0>;	/* CE0 */
 				reset-gpios = <&gpio 5 0>;
-				power-gpios = <&gpio 3 0>,
-				              <&gpio 7 0>;
+				power-gpios = <&gpio 23 0>,
+				              <&gpio 24 0>;
 				spi-irq-gpios = <&gpio 25 0>;
 				spi-max-frequency = <50000000>;
 				status = "okay";